[OSDOCS#18857]: Added a new module for Fencing Validator script (#110148)
openshift/openshift-docs: modules/Installation-fencing-validator-faq.adoc

> :_mod-docs-content-type: CONCEPT

Reviewer: This reads more like a reference file. See [4.1.3. Reference modules](https://redhat-documentation.github.io/modular-docs/#creating-reference-modules)

[id="installation-fencing-validator-script-faq_{context}"]

> = Frequently asked questions for fencing validator script

Reviewer: FAQ sections are not typical. Please check with your DPM or CS if we support such a section.

[role="_abstract"]

> The following section provides answers to some of the frequently asked questions:

Reviewer: Same question as the abstract should not be a lead-in sentence. The abstract should cover the what and why.

- **Can the script fix problems it finds?** +

> No. It is a diagnostic tool only. It reports what is wrong but does not attempt repairs.

Reviewer: ```suggestion
No. The script is a diagnostic tool only and reports what is wrong but does not attempt repairs.
```

- **Is it safe to run on a production cluster?** +

> Without `--disruptive`, yes. The non-disruptive checks are read-only. With `--disruptive`, it will reboot nodes, so we suggest to only be run during maintenance windows.

Reviewer: ```suggestion
Without `--disruptive`, yes. The non-disruptive checks are read-only. If the script use the `--disruptive` flag, the script reboots the nodes. Red Hat recommends to only use the flag during maintenance windows.
```

> Without `--disruptive`, yes. The non-disruptive checks are read-only. If the script uses the `--disruptive` flag, the script reboots the nodes. Red Hat recommends to use the flag only during maintenance windows.

Reviewer: ```suggestion
Without `--disruptive`, yes. The non-disruptive checks are read-only. If the script uses the `--disruptive` flag, the script reboots the nodes. Red Hat recommends using the flag only during maintenance windows.
```

- **Does the customer need to install it?** +

> No. It is automatically deployed by MCO on Two-Node with Fencing clusters. It is already on the nodes.

Reviewer: ```suggestion
No. The script gets automatically deployed by MCO on Two-Node with Fencing clusters. The script already exists on the nodes.
```

- **Can it be run from outside the cluster?** +

> The script itself lives on the nodes and runs there. You access it via `oc debug node/...`. If you have direct SSH access, you can also SSH to a node and run it directly.

Reviewer: ```suggestion
The script itself lives on the nodes and runs there. You can access the script by tusing the `oc debug node` command. If you have direct SSH access, you can also SSH to a node and run it directly.
```

- ** Does it work on Two-Node with Arbiter clusters?** +

> No. It is only installed on Two-Node with Fencing clusters. Arbiter clusters use a different quorum mechanism and do not use STONITH fencing.

Reviewer: ```suggestion
No. The script is installed only on Two-Node with Fencing clusters. Arbiter clusters use a different quorum mechanism and do not use STONITH fencing.
```
